pipelines/concourse.yml

```diff
@@ -1104,6 +1104,28 @@ jobs:
     - get: bbr-sdk-release
     - get: prod
     - get: ci
+  - put: bosh-topgun-worker-deployment
+    tags: [topgun]
+    params:
+      manifest: cbd/cluster/external-worker.yml
+      stemcells:
+      - gcp-xenial-stemcell/*.tgz
+      releases:
+      - concourse-release/*.tgz
+      - bpm-release/*.tgz
+      ops_files:
```

Comment:
I suspect that the `bosh-deployment-resource` is gonna expect this to be an array. Can you drop the param completely?

Reply:
Yup, that was a mistake even having it there.